js/acquia-lift-js-wrapper-min.js

```diff
@@ -1 +0,0 @@
r r,i,s,u,a;t==null&&(t={}),n==null&&(n=null),u=[e,"decisions"],r=null;if(t.choices!=null){s=t.choices;for(i in s){a=s[i];if((a!=null?a.join:void 0)==null)continue;u.push(i+":"+a.join(",")),r==null&&(r={}),r[i]||(r[i]={code:a[0]})}delete t.choices}return t.fallback!=null&&(r=t.fallback,delete t.fallback),this.send(u,t,null,!0,function(e){return function(t){var i,s;o(e.opts,t!=null?t.session:void 0);if(n==null)return;return s=(i=t!=null?t.decisions:void 0)!=null?i:r,n(s,t!=null?t.session:void 0)}}(this))},a.prototype.goal=function(e,t,n){var r;return r=[e,"goal"],t.goal!=null&&(r.push(t.goal),delete t.goal),this.send(r,t,null,!0,function(e){return function(t){var r,i;o(e.opts,t!=null?t.session:void 0);if(n==null)return;return r=(t!=null?t.reward:void 0)>0,i=(t!=null?t.agent:void 0)==null,n(r,t!=null?t.session:void 0,i)}}(this))},a.prototype.send=function(e,t,n,s,o){var a;t.apikey=this.apikey,this.opts.session!=null&&(t.session=this.opts.session),t._t=(new Date).getTime();if(!s||(a=this.opts.batching)!=="auto"&&a!=="manual")return e=this.opts.server+"/"+this.owner+"/"+e.join("/")+"?"+u(t),new this.opts.transport(e,n,this.opts.timeout,function(e){return function(e){var t,n;try{return n=JSON.parse(e),o(n)}catch(r){return t=r,o(null)}}}(this));this.batch.push(r(e,t,o)),this.opts.batching==="auto"&&i(this);return},u=function(e){var t,n,r;n="";for(t in e)r=e[t],n+="&"+t+"="+escape(r);return n},o=function(e,t){if(t!=null&&e.cookies!=null)return e.session=t,typeof e.scodestore=="function"?e.scodestore("mpid",t,e.cookies):void 0},s=function(e,t){var r;return r=null,function(){var i;return i=1<=arguments.length?n.call(arguments,0):[],clearTimeout(r),setTimeout(function(e){return function(){return t.apply(e,i)}}(this),e)}},a.prototype.batchStart=function(){return this.batch=[]},a.prototype.batchSend=function(){var e,t,n;n=["-","batch"],e=this.batch.concat(),this.batchStart();if(e.length>0)return t=this.opts.batchingSkipsPreflight?{_type:"json"}:{},this.send(n,t,e,!1,function(t){var n,r,i;i=[];for(n in e)r=t!=null?t[n]:void 0,i.push(e[n].cb(r!=null?r.data:void 0));return i});return},i=s(20,function(e){return e.batchSend()}),r=function(e,t,n){var r;r={agent:e[0],type:e[1],query:t,cb:n};switch(r.type){case"decisions":e.length>2&&(r.choices=e.slice(2).join("/"));break;case"goal":e.length>2&&(r.goal=e[2])}return r},a}(),e=function(){function t(t,n,r,i){var s;s=e(t,n);if(s==null)return i(null);typeof r=="number"&&(s.timeout=r),s.onload=function(){return i(s.responseText)},s.onerror=s.ontimeout=function(){return i(null)},n!=null?(t.indexOf("_type=json")>0||s.setRequestHeader("Content-Type","application/json"),s.send(JSON.stringify(n))):s.send()}var e;return e=function(e,t){var n,r;return n=t!=null?"POST":"GET",typeof XMLHttpRequest!="undefined"&&XMLHttpRequest!==null&&(r=new XMLHttpRequest),(r!=null?r.withCredentials:void 0)!=null?r.open(n,e,!0):typeof XDomainRequest!="undefined"&&XDomainRequest!==null?(r=new XDomainRequest,r.open(n,e)):r=null,r},t}()}).call(this);
```

Comment:
Yay - I think that means there is some outdated documentation somewhere about how to generate this.